test/NuGet.Core.Tests/NuGet.Build.Tasks.Test/GetRestoreProjectStyleTaskTests.cs

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using FluentAssertions;
+using NuGet.Configuration;
+using NuGet.ProjectModel;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.Build.Tasks.Test
+{
+    public class GetRestoreProjectStyleTaskTests
+    {
+        [Fact]
+        public void GetRestoreProjectStyleTask_CanDetectPackageReference()
```

Comment:
Rename tests with the following format:  `<MemberName>_<Condition>_<Expectation>`.  I know some older tests don't follow this pattern.

Example:

    ProjectStyle_WhenProjectHasPackageReferenceItems_ReturnsPackageReference